frontend/src/pages/PromotionsPage/PromotionsPage.module.scss

```diff
@@ -0,0 +1,55 @@
+@import '/src/variables';
+@import '/src/mixins';
+
+.section {
+  padding: 16px 0 140px;
+
+  @include md {
+    padding: 24px 0 44px;
+  }
+
+  @include sm {
+    padding: 36px 0 44px;
+  }
+
+  &-title {
+    font-weight: 500;
+    font-size: 32px;
+    line-height: 41.6px;
+    margin-bottom: 38px;
+
+    @include lg {
+      margin-bottom: 38px;
+    }
+
+    @include md {
+      margin-bottom: 38px;
+    }
+
+    @include sm {
+      display: none;
+    }
+  }
+}
+
+.promotion-cards {
```

Comment:
check the design, there are also hover properties for the cards

Reply:
fixed